backend/test/Test/Edna.hs

```diff
@@ -1,12 +1,18 @@
 module Test.Edna
   ( unit_stub_is_2
+  , spec_db
   ) where
 
 import Universum
 
+import Test.Hspec (Spec, it, shouldBe)
```

Comment:
Oh, I think we should use either `tasty` or `hspec` but not both. I previously tried both and I think both are acceptable, I liked `tasty` slightly more, but `hspec` is also ok. One advantage of `tasty` is that it can generate XML reports that are rendered nicely by GitLab, but here we are using GitHub, so it doesn't matter.

If you strictly prefer `hspec`, let's use it, then remove `tasty` completely. If you are ok with `tasty`, let's use it and not use `hspec`. Using both is redundant.